src/client/app/strings/english.ts

```diff
@@ -40,6 +41,7 @@ export function getEnglishStrings(): ClientStrings {
         snippetImportErrorBody: `We couldn't import the snippet.`,
         reloadPrompt: 'Reload this task pane and then try another URL or ID.',
 
+        snippetNameDuplicationError: 'There is an existing snippet with this name. Do you want to go to this snippet or create a new one with a different name?',
```

Comment:
How about "There is an existing snippet that was imported from the same Gist ID.  Do you want to switch to this existing snippet, or create a new copy?"